### PR TITLE
refactor: Add protocol versions to "About UniMe" page

### LINKED ISSUE #265: Add protocol versions to "About Me" page
This has been added to the About page in dev mode. Still needs to be added to the README. 

Most recent versions: https://github.com/impierce/openid4vc?tab=readme-ov-file.

### PR DESCRIPTION
**Pull request by maiertech**
# Description of change

List the specs and their versions used in UniMe's about page but only in dev mode.

## Links to any relevant issues

Fixes #265.

## How the change has been tested

Visual check of the "About UniMe" page (turn dev mode on and off).

## Definition of Done checklist

Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by maiertech** [comment]
Display in dev mode only.

**Review by nanderstabel — DISMISSED** [review]
Looks good! Except that in desktop mode clicking one of the links result in this page:
image

It's really not a big deal since we're the only once using Desktop, but in case it is a quick fix it might be worth it to fix it.

**Comment by nanderstabel** [comment]
> Looks good! Except that in desktop mode clicking one of the links result in this page: image
> 
> It's really not a big deal since we're the only once using Desktop, but in case it is a quick fix it might be worth it to fix it.

this is the corresponding error in my terminal:
```
/usr/lib/x86_64-linux-gnu/webkit2gtk-4.1/WebKitNetworkProcess: symbol lookup error: /snap/core20/current/lib/x86_64-linux-gnu/libpthread.so.0: undefined symbol: __libc_pthread_init, version GLIBC_PRIVATE
```

**Comment by nanderstabel** [comment]
The WebKit issue I encountered is related to my VSCode somehow. After following [these](https://stackoverflow.com/a/75956168) instructions the problem was resolved for me.

**Review by nanderstabel — DISMISSED** [review]

File: unime/src/routes/(app)/me/settings/about/+page.server.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+interface Specification {
+  id: string;
+  description: string;
+  version: string;
+  url: string;
+}
+
+export const load = (async () => {
+  // https://github.com/impierce/openid4vc?tab=readme-ov-file
```
Comment (daniel-mader) [inline unime/src/routes/(app)/me/settings/about/+page.server.ts]: Link is broken

File: unime/src/routes/(app)/me/settings/about/+page.server.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  const specifications: Specification[] = [
+    {
+      id: 'OID4VCI',
+      description: 'OpenID for Verifiable Credential Issuance',
+      version: 'Working Group Draft 13 published: 8 February 2024',
```
Comment (daniel-mader) [inline unime/src/routes/(app)/me/settings/about/+page.server.ts]: I would add some separation between the version and its publishing date, such as:
* `Working Group Draft 13, published: 8 February 2024`
* `Working Group Draft 13 (published: 8 February 2024)`

File: unime/src/routes/(app)/me/settings/about/+page.svelte [hunk 3]
```diff
@@ -3,24 +3,47 @@
...
 <TopNavBar on:back={() => history.back()} title={$LL.SETTINGS.SUPPORT.ABOUT.NAVBAR_TITLE()} />
+
 <div class="content-height flex flex-col bg-silver dark:bg-navy">
-  <div class="flex flex-col p-8">
-    <!-- Footer -->
-    <div
-      class="flex flex-col items-center pt-4 text-[13px]/[24px] font-normal text-slate-500 opacity-50 dark:text-slate-300"
-    >
-      <!-- <div class="pb-4" /> -->
+  <h1 class="hidden">{$LL.SETTINGS.SUPPORT.ABOUT.TITLE}</h1>
```
Comment (daniel-mader) [inline unime/src/routes/(app)/me/settings/about/+page.svelte]: Why does this have a class "hidden"?

Comment (daniel-mader) [reply unime/src/routes/(app)/me/settings/about/+page.svelte]: Also, `TITLE` should be `TITLE()` to resolve from `typesafe-i18n`.

File: unime/src/routes/(app)/me/settings/about/+page.server.ts (hunk 2 above)
Comment (maiertech) [reply unime/src/routes/(app)/me/settings/about/+page.server.ts]: () added.

File: unime/src/routes/(app)/me/settings/about/+page.server.ts (hunk 1 above)
Comment (maiertech) [reply unime/src/routes/(app)/me/settings/about/+page.server.ts]: Fixed.

File: unime/src/routes/(app)/me/settings/about/+page.svelte (hunk 3 above)
Comment (maiertech) [reply unime/src/routes/(app)/me/settings/about/+page.svelte]: Route did not have `h1`. But should always have `h1` for accessiblity.

Changed to `sr-only`. `hidden` is indeed ignored by screen readers.

**Review by daniel-mader — APPROVED** [review]

**Review by nanderstabel — APPROVED** [review]

**Pull request merged**